Move and rename security packages

arjantijms opened the pull request:
Rename main package in security-ee module to include .ee

Before the security and security-ee modules both had the same main package name.

Rename auth package to authentication in .ee package, and move the GlassFish LoginModule/Realms combinations to a new package.

Rename JDBCDigest which had nothing to do with JDBC.

arjantijms committed:
Rename main package in security-ee module to include .ee

Before the security and security-ee modules both had the same main
package name.

Rename auth package to authentication in .ee package, and move the
GlassFish LoginModule/Realms combinations to a new package.

Rename JDBCDigest which had nothing to do with JDBC.

Signed-off-by: Arjan Tijms <arjan.tijms@omnifish.ee>

arjantijms committed:
Update test packages

Signed-off-by: Arjan Tijms <arjan.tijms@omnifish.ee>

arjantijms committed:
Remove accidentally committed annotation

Signed-off-by: Arjan Tijms <arjan.tijms@omnifish.ee>

arjantijms commented on appserver/tests/admin/tests/src/test/java/org/glassfish/main/admin/test/ClusterITest.java:
```diff
@@ -60,6 +61,7 @@
  * @author Tom Mueller
  */
 @TestMethodOrder(OrderAnnotation.class)
+@Disabled
```
Good that you found it, that should not have been committed.

avpinchuk commented on appserver/security/core-ee/src/main/java/com/sun/enterprise/security/ee/SecurityContainer.java:
```diff
@@ -26,16 +26,15 @@
  *
  */
-@Service(name = "com.sun.enterprise.security.ee.SecurityContainer")
+@Service(name = "com.sun.enterprise.security.ee.ee.SecurityContainer")
```
@arjantijms, it looks like the some bad thinh - double `ee` in package name

arjantijms replied:
Thanks, fixed it!

avpinchuk commented on appserver/security/core-ee/src/main/java/com/sun/enterprise/security/ee/SecuritySniffer.java:
```diff
@@ -41,7 +41,7 @@
 public class SecuritySniffer extends GenericSniffer {
 
-    final String[] containers = { "com.sun.enterprise.security.ee.SecurityContainer" };
+    final String[] containers = { "com.sun.enterprise.security.ee.ee.SecurityContainer" };
```
Here too

arjantijms committed:
Fixed global replace error

Signed-off-by: Arjan Tijms <arjan.tijms@omnifish.ee>

arjantijms committed:
Temp fix for mysterious import of "com.sun.enterprise.security.web"

It's not clear where it comes from, but excluding it helps. Should find
root cause before merging.

Signed-off-by: Arjan Tijms <arjan.tijms@omnifish.ee>

avpinchuk commented:
@arjantijms, `appserver/web/web-glue` contains `com.sun.enterprise.security.web` package. Maybe cause here?

arjantijms commented:
> @arjantijms, `appserver/web/web-glue` contains `com.sun.enterprise.security.web` package. Maybe cause here?

Might be, the problem seems to be the other way around. It manually exports packages `com.sun.enterprise.security.web` was globally renamed to `com.sun.enterprise.security.ee.web`, but there it should not be renamed.

arjantijms committed:
Fix export of com.sun.enterprise.security.web package

Signed-off-by: Arjan Tijms <arjan.tijms@omnifish.ee>

avpinchuk commented on appserver/security/core-ee/src/test/resources/com/sun/enterprise/security/ee/perms/server.policy:
```diff
@@ -99,7 +99,7 @@ grant {
 // Permissions to invoke CORBA objects in server
 grant {
-    permission com.sun.enterprise.security.CORBAObjectPermission "*", "*";
+    permission com.sun.enterprise.security.ee.CORBAObjectPermission "*", "*";
```
Fix, please here

arjantijms replied:
Thanks!

avpinchuk commented on appserver/security/core-ee/pom.xml:
```diff
@@ -213,7 +213,7 @@
                     <archive>
                         <manifestEntries>
-                            <probe-provider-class-names>com.sun.enterprise.security.auth.realm.RealmsProbeProvider,com.sun.enterprise.security.WebSecurityDeployerProbeProvider,com.sun.enterprise.security.EjbSecurityPolicyProbeProvider</probe-provider-class-names>
+                            <probe-provider-class-names>com.sun.enterprise.security.ee.auth.realm.RealmsProbeProvider,com.sun.enterprise.security.ee.WebSecurityDeployerProbeProvider,com.sun.enterprise.security.ee.EjbSecurityPolicyProbeProvider</probe-provider-class-names>
```
Please, view here. You do not touch nucleus classes

arjantijms committed:
Correct wrongly renamed class references

Signed-off-by: Arjan Tijms <arjan.tijms@omnifish.ee>
